microdnf/commands/swap/swap.hpp

```diff
@@ -23,22 +23,24 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 
 #include <libdnf-cli/session.hpp>
-#include <libdnf/conf/option_bool.hpp>
 
-#include <memory>
 #include <vector>
 
 
 namespace microdnf {
 
 
+// TODO(jrohel): The "swap" command may be removed in the future in favor of a more powerful command (eg "do"),
```

Comment:
Minor - I am not 100% sure but there was a suggestion to name it `perform`. If you agree you can already to add it into note.